src/Symfony/Component/Ldap/Adapter/EntryManagerInterface.php

```diff
@@ -40,4 +40,13 @@ public function update(Entry $entry);
      * @param Entry $entry
      */
     public function remove(Entry $entry);
+
+    /**
+     * Renames an entry on the Ldap server.
+     *
+     * @param Entry  $entry
+     * @param string $newRdn
+     * @param bool   $removeOldRdn
+     */
+    public function rename(Entry $entry, $newRdn, $removeOldRdn = true);
```

Comment:
Maybe a `MoveEntryInterface` would be a good solution?

```php
interface MoveEntryInterface
{
    public function move(Entry $entry, $newRoot);
}
```
Which should be merged in the main interface for Symfony 4.0.